sunpy/net/tests/test_fido.py

```diff
@@ -229,6 +230,17 @@ def test_vso_unifiedresponse():
     assert isinstance(uresp, UnifiedResponse)
 
 
+@pytest.mark.remote_data
+def test_vso_error():
+    with pytest.warns(SunpyUserWarning,
+        match="VSO-D404 Bad Request -- Invalid Time \
+Sample : contains non digit"):
```

Comment:
I don't think this line continuation, can you reformat this please?

Reply:
Done